app/javascript/src/js/recent_items_tooltips.js

```diff
@@ -0,0 +1,3 @@
+$( document ).ready(function() {
+    jQuery('[data-toggle="tooltip"]').tooltip();
+});
```

Comment:
there's nothing about this that is specific to recent_items, right? It's activating everything with data-toggle tooltip. 

In fact, I'm surprised we aren't already doing this somewhere else? Weren't we already using tooltips?  Oh, I see, ohms_footnotes.js does it in a... customized way. 

The ohms_footnotes.js code is also triggered simply on `jQuery('[data-toggle="tooltip"]')`, but then does custom setup involving container and delay -- I think we want to be more careful these two areas don't collide with each other -- they're both going to try to operate on all data-toggle=tooltips, doing different things -- and they are both loaded on all pages!

Either the ohms one can be triggered by `data-toggle=ohms-tooltip`, or it should apply to only tooltips within an ohms container, or something. 

And generic `jQuery('[data-toggle="tooltip"]').tooltip();` should be in a file just called `js/bootstrap_tooltips_activate` or something, as it is not at all restriccted to recent items.

Reply:
Yup -- narrowed this down so it is only applied to the recent items. (Code is still executed on all pages, though.)

Reply:
@eddierubeiz the problem is that the existing JS for OHMS will still apply to recent items too.... the recent items still get both JS's trying to apply to them. 

I think you're going to need to put something other than data-toggle=tooltip on *one* of them to avoid having two sets of code apply to one of them. Since this is the one that is a totally generic tooltip, I'd leave this one data-toggle=tooltip, and put something on the OHMS taht makes this one *not* apply to it, only the special OHMS tooltip setup.

Reply:
Good call . The `ohms-tooltip` is a great idea in particular.

Reply:
Maybe even move em both to the same JS file, since they have such possibility to potentially conflict with each other, they could be right next to each other so the dev can see that, instead of needing to remember "oh yeah I did other tooltips in that ohter file taht might conflict..."

Reply:
Hmmmmmmm yeah good idea.